.github/workflows/main-ci.yml

```
@@ -0,0 +1,707 @@
+          echo "Please review the dry run output. If ready, manually trigger this workflow again with:" >> $GITHUB_STEP_SUMMARY
+          echo "- 'Run NPMJS.org publish pipeline' checked" >> $GITHUB_STEP_SUMMARY
+          echo "- 'CONFIRM: Actual publish to NPMJS.org' checked" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          [ -f CHANGELOG.md ] && cat CHANGELOG.md >> $GITHUB_STEP_SUMMARY || echo "CHANGELOG.md not found" >> $GITHUB_STEP_SUMMARY
+
+  publish_to_npmjs_org:
+    name: Publish to NPMJS.org (Confirmed)
+    needs: [prepare_and_dry_run_npmjs]
+    runs-on: ubuntu-latest
+    # CRITICAL CONDITION: Only run if manually dispatched with both flags set
+    if: |
```

actionlint (1.7.7)

got unexpected character '#' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)